### DISCUSSION TITLE
Question(s)

### DISCUSSION THREAD

**Discussion by Fireworkstars46** [AdvancedXRay/XRay-Mod#241] [Q&A]
As for the 1.6.4 version of this mod, would I be able to add more ores to the config?

**Comment by MichaelHillcox** [comment]
Yes? Not sure what you're really asking here

**Comment by Fireworkstars46** [comment]
Exactly what I'm asking. Add more ores etc to the config. Like I have a mod called "railcraft" and I want to add their saltpeter ore spawner to it

**Reply by MichaelHillcox** [reply]
Then just add it via the GUI?

**Reply by Fireworkstars46** [reply]
Does the 1.6.4 version have a gui?
